ttnn/cpp/ttnn/operations/core/to_layout/to_layout_op.cpp

```diff
@@ -109,7 +110,11 @@ Tensor to_layout_impl(
 
     auto padded_output_shape = output_shape;
     for (auto index = output_shape.size() - 2; index < output_shape.size(); ++index) {
-        padded_output_shape[index] = ttnn::pad_to_multiple_of_tile_size(padded_output_shape[index]);
+        if (index == output_shape.size() - 2) { // h
```

Comment:
Here and all the changes below in this file, assuming the variables are small or properly referenced, it's better to do
```
a = ...? a1 : a0;
f(a);
```
or
```
f(...? a1 : a0);
```
or
```
a;
if () {
 a=...
} ... else {
 a=...
}
f(a)
```
instead of
```
if () {
  f(a1);
} ... else {
  f(a0);
}
```

Please change if possible.

Reply:
changed